features/speech-recognition.yml

```
@@ -0,0 +1,45 @@
+name: Speech recognition
+description: The `SpeechRecognition` API converts audio into text using the device's speech recognition service, providing methods and events for fine-grained control and observation of the interaction.
```

I think we can go more brief here and dodge having to make observation or interaction more meaningful.

```suggestion
description: The `SpeechRecognition` API converts audio into text using the device's speech recognition service.
```

---

I always to include a mention when there are events. Not a blocker and we can come up w/ general approaches to this and modify later if we think is needed.